Source/Assimil8or/Assimil8or.cpp

```diff
@@ -498,6 +498,228 @@ void Assimil8orValidator::sortContentsOfFolder (juce::ValueTree folderVT)
 
 Assimil8orPreset::Assimil8orPreset ()
 {
+    std::function<int()> getParameterIndex = [this] ()
+    {
+        return key.fromFirstOccurrenceOf (" ", false, false).getIntValue ();
+    };
+    
+    // Global Action
+    actions.insert({
```

Comment:
I think we should go back to having action maps per section. The current single map works now because none of the sections have parameters of the same name. But, a future release of the eurorack module could add a parameter to a section, which also exists in another section.